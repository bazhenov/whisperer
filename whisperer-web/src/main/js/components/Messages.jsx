import React from 'react'
import Message from './Message.jsx'

export default ({ messages }) =>
	<div className="row">
		<div className="col-md-12">
			{ messages.map((m, key) => <Message message={m} key={key + '_' + m.timestamp} />) }
		</div>
	</div>
