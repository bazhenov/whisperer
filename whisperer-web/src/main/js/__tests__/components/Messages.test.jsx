import React from "react";
import Messages from "../../components/Messages.jsx";
import renderer from "react-test-renderer";
import { message, message2 } from '../fixtures'

it('Message', () => {
	expect(renderer.create(
		<Messages messages={[message, message2]} />
	).toJSON()).toMatchSnapshot()
});