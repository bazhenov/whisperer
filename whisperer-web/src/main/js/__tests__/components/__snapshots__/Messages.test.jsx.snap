// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message 1`] = `
<div>
  <div
    className="message-row"
  >
    <div
      className="message-head"
    >
      <div
        className="message-time"
      >
        14:17:09.683
      </div>
      <div
        className="message-host"
      >
        @
        http://search-coord5.akod.loc
      </div>
      <div
        className="message-group"
      >
        <div
          className="message-tooltip"
        >
          <div>
            c.f.s.CoordinatorSearchEngine
          </div>
          <span
            className="message-tooltip-text"
          >
            com.farpost.search.CoordinatorSearchEngine
          </span>
        </div>
      </div>
      <div
        className="message-thread"
      >
        qtp-12345
      </div>
    </div>
    <div
      className="message-content"
    >
      <span>
        Delegating execution to command 
      </span>
      <span
        className="argument-value"
      >
        StagedSearchQuery
      </span>
      <span>
        , query: '
      </span>
      <a
        href="#"
        onClick={[Function]}
      >
        ftext contains('s
        ...
      </a>
      <span>
        ', sort: 
      </span>
      <span
        className="argument-value"
      >
        null
      </span>
      <span>
        , facets: 
      </span>
      <span
        className="argument-value"
      >
        directoryId
      </span>
      <span>
        , filter: '
      </span>
      <a
        href="#"
        onClick={[Function]}
      >
        siteId!='1632' an
        ...
      </a>
      <span>
        '
      </span>
    </div>
  </div>
  <div
    className="message-row"
  >
    <div
      className="message-head"
    >
      <div
        className="message-time"
      >
        14:17:09.708
      </div>
      <div
        className="message-host"
      >
        @
        http://search-node68.akod.loc
      </div>
      <div
        className="message-group"
      >
        <div
          className="message-tooltip"
        >
          <div>
            c.f.s.i.IndexNodeImpl
          </div>
          <span
            className="message-tooltip-text"
          >
            com.farpost.search.index.IndexNodeImpl
          </span>
        </div>
      </div>
      <div
        className="message-thread"
      >
        qtp1914572623-35
      </div>
    </div>
    <div
      className="message-content"
    >
      <span>
        Search issued: 
      </span>
      <a
        href="#"
        onClick={[Function]}
      >
        +(+(+system.defin
        ...
      </a>
      <span>
        
      </span>
    </div>
  </div>
</div>
`;
